src/ScriptBuilder/Saga/AllSagaDefinitionReader.cs

```diff
@@ -27,12 +29,15 @@ public IEnumerable<SagaDefinition> GetSagas(Action<ErrorsException, TypeDefiniti
             }
             catch (ErrorsException exception)
             {
-                logError(exception, type);
+                errors.Add(new Exception($"Error in '{type.FullName}' (Filename='{type.GetFileName()}'). Error: {exception.Message}", exception));
             }
         }
-        return sagas;
-    }
-
 
+        if (errors.Count > 0)
+        {
+            throw new AggregateException(errors);
```

Comment:
The way we used to do this, we would still generate a partial set of scripts if there were errors. i.e. If you had 5 sagas and one of them failed, you would get 4 scripts _before_ the error was thrown.

Reply:
I reverted back this error logging changes, so when used in MSBuild it will have the same old behavior.